source/_integrations/homekit_controller.markdown

```diff
@@ -83,13 +83,34 @@ HomeKit controller will poll your devices, but it will also automatically enable
-
-HomeKit Controller will automatically discover supported Thread devices using the [Zeroconf](/integrations/zeroconf) integration. Battery powered devices may go to sleep and require a button pressing to wake them up before pairing works.
-
-In order to provision a Thread device onto a mesh network using a HomePod as a border router, first pair it with an iOS device. Then unpair it from the Home app. Don't reset it. This will leave the Thread network details on the device. Home Assistant can now pair with it over Thread.
+## Adding a HomeKit controller device to a Thread network
+
+### Prerequisites
+
+* Make sure your HomeKit device supports Thread. This is indicated by the Thread label on the packaging.
+* **Bluetooth support**: HomeKit devices are joined using Bluetooth. If your Home Assistant server device does not support Bluetooth, you can use a Bluetooth proxy.
+* **Thread network**: In order to use HomeKit over Thread you need a working border router. 
+    * Make sure your Home Assistant device is on the same network (LAN) as the border router. 
+    * If you have a Home Assistant Yellow or SkyConnect, you can enable multiprotocol to set up an Open Thread border router and with that a Thread network. 
+      * Documentation on [enabling multiprotocol on Yellow](https://yellow.home-assistant.io/guides/enable-multiprotocol/)
+      * Documentation on [enabling multiprotocol on SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-multiprotocol/)
+    * Other devices, such as an Apple HomePod mini or Apple TV, cannot currently be used as border routers. They do not communicate their Thread credentials an can therefore not be integrated.
+
+### To add a HomeKit controller device to a Thread network
+
+1. Power up your HomeKit controller device.
+   * If you have Bluetooth enabled, it should be discovered under **{% my integrations title="Settings > Devices & Services" %}**.
+1. On the HomeKit integration, select **Configure**.
+1. To pair the device, enter the HomeKit pairing code. The code is on the device itself or the packaging. 
+   * Select **Finish**. 
+1. To open the device configuration page, on the integration, select the **device**.
+1. Under **Diagnostic**, you can see the **Thread Status** as **Disabled**.
+    ![Device configuration page](/images/integrations/homekit_controller/homekit_controller_add_02.png)
+1. To enable Thread, under **Configuration**, select **Press**. This will provision the preferred Thread credentials.
+   * The status has now changed to **Child**. 
+     ![Thread status](/images/integrations/homekit_controller/homekit_controller_add_02.png)
+   * That's it. Your controller device now communicates via Thread.
+   * Your HomeKit controller automatically discovers supported Thread devices using the [Zeroconf](/integrations/zeroconf) integration. Battery powered devices may go to sleep. To pair them, you may need to press a button to wake them up.
```

Comment:
This is related to the other method to on-board a Thread device, so it doesn't belong here.

```suggestion
```